### DISCUSSION TITLE
No Options Available for Public, Pre-indexed rRNA Databases

### DISCUSSION THREAD

**Discussion by robmcman** [biocore/sortmerna#380] [Q&A]
Hello! 

I am attempting to follow a tutorial on how to perform a [metatranscriptome](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/metatranscriptomics/tutorial.html) analysis and got stuck. I was trying to input my QC controlled reads into sortmerna for rRNA removal on Galaxy, but what I saw diverged from the tutorial. Specifically, there are no no options available for public, pre-indexed rRNA databases and, I try to run it without one, I receive the following error: 
"
/usr/local/bin/merge-paired-reads.sh: line 54: perl: command not found
/usr/local/bin/merge-paired-reads.sh: line 56: perl: command not found
/usr/local/bin/merge-paired-reads.sh: line 58: paste: command not found

  [0;31mERROR[0m: the file  could not be opened:  No such file or directory.
"
Additionally,[ here](https://usegalaxy.org/u/robbbbb/h/practice-metatranscriptomics) is my workflow.

Am I doing something wrong? Any help would be appreciated. Thanks!

**Comment by cmith44** [comment]
@robmcman Closing this conversation as the question was addressed in #381.
